src/components/Roadmap.js

```diff
@@ -57,6 +57,12 @@ const Roadmap = () => {
     implemented: blankIssues,
   })
 
+  const isMerged=async(url)=>{
+    const resp= await axios.get(url)
+    const data = await resp.data
+return ( data.merged===true)
```

Comment:
if `data.merged` returns a boolean, we don't need the `===true`... if if doesn't return a boolean, can use `return !!data.merged` to return a boolean from it.

Reply:
And help me out here... do we not already have this data available from the axios call within the `useEffect` hook? Not sure I understand why we're making a separate request. Mind walking me through your thought process here?